### PR TITLE
Show sites with blank titles and add hints for deleted sites on the themes and plugins pages

### PR DESCRIPTION
**Pull request by hvianna**
Hello,

I've made a small modification to display the site path for sites with blank titles, so they show properly on the network themes and plugins pages. 

Thanks for the great plugin!

### CODE REVIEW

**Comment by hvianna** [comment]
Hello again! :)

I just added another commit to add hints for deleted sites, and also to allow the `class` attribute in the `li` elements, for proper styling of archived and deleted sites.

I hope you find the changes appropriate. Best regards!

File: inc/autoload/class-add-plugin-list.php [hunk 1]
```diff
@@ -209,9 +210,13 @@ public function manage_plugins_custom_column( $column_name, $plugin_file, $plugi
 						$class = ' class="site-archived"';
 						$hint  = esc_attr__( ', Archived site', 'multisite-enhancements' );
 					}
+					elseif ( $this->is_deleted( $key ) ) {
+						$class = ' class="site-deleted"';
+						$hint  = esc_attr__( ', Deleted site', 'multisite-enhancements' );
```
Comment (bueltge) [inline inc/autoload/class-add-plugin-list.php]: The comma on start is hard for translators, so please add in minimum the hint for the translators. See https://gist.github.com/bueltge/49c652187ef10a448bfb#you-want-to-output-the-username-in-a-sentence

File: inc/autoload/class-add-plugin-list.php [hunk 2]
```diff
@@ -209,9 +210,13 @@ public function manage_plugins_custom_column( $column_name, $plugin_file, $plugi
...
+					}
 					$output .= '<li' . $class . ' title="Blog ID: ' . $key . $hint . '">';
 					$output .= '<nobr><a href="' . get_admin_url( $key ) . 'plugins.php">'
-					. $value['name'] . '</a>' . $hint . '</nobr></li>';
+					. ( trim( $value['name'] ) ?: $value['path'] ) . '</a>' . $hint . '</nobr></li>';
```
Comment (bueltge) [inline inc/autoload/class-add-plugin-list.php]: The nobr tag is deprecated in html, maybe we should use this PR as chance to replace them with span and css.

File: inc/autoload/class-add-plugin-list.php (hunk 1 above)
Comment (hvianna) [reply inc/autoload/class-add-plugin-list.php]: I would rather leave the comma out of the translated string, like
```
$hint  = ', ' . esc_attr__( 'Archived site', 'multisite-enhancements' );
```
this way the structure won't be broken by a translator mistake.

And how about we use the native WP translations for archived and deleted, so they remain consistent with other areas of the admin dashboard (screenshot below). What do you think?

image

While I was testing it, I realized a site can be both deleted and archived, so I'll change that `elseif` statement.

File: inc/autoload/class-add-plugin-list.php (hunk 2 above)
Comment (hvianna) [reply inc/autoload/class-add-plugin-list.php]: Sure! Done in https://github.com/bueltge/wordpress-multisite-enhancements/pull/62/commits/5a927603d73209bcd3b21b26871ccef7ecf15c90

File: inc/autoload/class-add-plugin-list.php (hunk 1 above)
Comment (bueltge) [reply inc/autoload/class-add-plugin-list.php]: The native version would be really fine, god idea. So we need only the comma outside the string for translation. However, I'm not sure there are languages there always use the comma for the arrangement of strings.
Thanks also for the hints, good part of testing.

**Comment by bueltge** [comment]
In the context of removing `nobr` tag from `wp_kses` is also necessary to change them on other pages - https://github.com/bueltge/wordpress-multisite-enhancements/search?q=%3Cnobr%3E&unscoped_q=%3Cnobr%3E 
Two entries there need a `span` tag.

**Comment by hvianna** [comment]
> 
> 
> In the context of removing `nobr` tag from `wp_kses` is also necessary to change them on other pages - https://github.com/bueltge/wordpress-multisite-enhancements/search?q=%3Cnobr%3E&unscoped_q=%3Cnobr%3E
> Two entries there need a `span` tag.

Yes, I have already replaced them in that same commit.. https://github.com/hvianna/wordpress-multisite-enhancements/commit/5a927603d73209bcd3b21b26871ccef7ecf15c90

File: inc/autoload/class-add-plugin-list.php (hunk 1 above)
Comment (hvianna) [reply inc/autoload/class-add-plugin-list.php]: Cool, I'll commit these changes. WP already uses the comma to separate site statuses in the site list, so I think it's ok.

**Pull request merged**